staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -108,6 +108,14 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 		userInfo, _ := request.UserFrom(ctx)
 		staticAdmissionAttributes := admission.NewAttributesRecord(nil, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Update, userInfo)
 		admissionCheck := func(updatedObject runtime.Object, currentObject runtime.Object) error {
+			isNotZeroObject, err := hasUID(currentObject)
+			if err != nil {
+				return fmt.Errorf("unexpected error when extracting UID from oldObj: %s", err.Error())
```

Comment:
we usually do `%v` and `err`

Reply:
`if isNotZeroObject, err := hasUID(currentObject); err != nil { ... } else if !isNotZeroObject { ... }`